packages/t/timeshift/package.yml

```diff
@@ -0,0 +1,28 @@
+name       : Timeshift
+version    : 24.01.1
+release    : 1
+source     :
+    - https://github.com/linuxmint/timeshift/archive/refs/tags/master.mint21.tar.gz : 7272f5007de6e831e55bbae8ffe1b5bd176c304c4a86f65ef62fb6cb08b556f3
+homepage   : https://github.com/linuxmint/timeshift/tree/master.mint21
```

Comment:
Trim this down to `https://github.com/linuxmint/timeshift`

Reply:
Done, thank you.